src/views/OtherStoresInventoryModal.vue

```diff
@@ -6,25 +6,73 @@
           <ion-icon :icon="close" />
         </ion-button>
       </ion-buttons>
-      <ion-title>{{ translate("Other stores inventory") }}</ion-title>
+      <ion-title>{{ translate("Other stores") }}</ion-title>
+      <ion-buttons slot="end">
+        <ion-button @click="sortByDistance">
+          <ion-icon slot="icon-only" :icon="locationOutline"
+            :color="activeSortType === 'distance' ? 'primary' : 'medium'"></ion-icon>
+        </ion-button>
+        <ion-button @click="sortByName">
+          <ion-icon slot="icon-only" :icon="textOutline"
+            :color="activeSortType === 'name' ? 'primary' : 'medium'"></ion-icon>
+        </ion-button>
+      </ion-buttons>
     </ion-toolbar>
+    <ion-searchbar v-model="queryString" :placeholder="translate('Search store name')" :debounce="500"
+      @ionInput="queryString = $event.target.value; searchFacilities()"
+      @keyup.enter="queryString = $event.target.value; searchFacilities()" />
   </ion-header>
-  <ion-content class="ion-padding">
-    <ion-searchbar v-model="queryString" @keyup.enter="queryString = $event.target.value; searchFacilities()"/>
-    <ion-list v-if="storesInventory.length">
-      <ion-item v-for="storeInventory in storesInventory" :key="storeInventory.facilityName">
-        <ion-label class="ion-text-wrap">{{ storeInventory.facilityName }}</ion-label>
-        <ion-note slot="end">{{ translate('ATP', { count: storeInventory.stock}) }}</ion-note>
+  <ion-content>
+    <ion-list>
+      <ion-item>
+        <ion-label>{{ translate("Hide facilities without stock") }}</ion-label>
+        <ion-toggle @click="toggleHideEmptyStock" />
       </ion-item>
     </ion-list>
+    <ion-accordion-group v-if="storesWithInventory.length">
+      <ion-accordion v-for="store in storesWithInventory" :key="store.storeCode" :value="store.storeCode"
+        :toggle-icon="hasWeekCalendar(store) ? chevronDownOutline : false" :readonly="!hasWeekCalendar(store)">
+        <ion-item slot="header" lines="inset">
+          <ion-label class="ion-text-wrap">
+            <p class="overline">{{ store.storeCode }}</p>
+            <h2>{{ store.storeName }}</h2>
+            <ion-note v-if="hasWeekCalendar(store)" :color="store.isOpen ? '' : 'danger'">
+              {{ store.hoursDisplay }}
+            </ion-note>
+            <p v-if="typeof store.dist === 'number'">{{ Math.round(store.dist) }} miles</p>
+          </ion-label>
+          <div slot="end">
+            <ion-note slot="end">{{ translate('ATP', { count: store.stock }) }}</ion-note>
```

Comment:
No need for slot=end on ion-note if used with div as div is already having end slot. Also is the div required, can't we directly define ion-note with end slot without div?